### PR TITLE
Add h-card classes and extra hidden metadata

### PR DESCRIPTION
**Pull request by Zegnat**
This is some work towards #63. (Although there was no Work post for it.) This makes the header on every page a parsable `h-card`.

* It _should_ also be accepted as [representative h-card](https://indieweb.org/representative_h-card) per the second step of [the discovery algorithm](http://microformats.org/wiki/representative-h-card-parsing). Though adding the explicit `u-uid` could be done if deemed necessary.
* Software Architect will come back as your role.
* Singapore is defined as country name. Not sure how addresses usually work there, maybe make it both country and city (`p-locality`)?
* Photo is added as invisible metadata.
* Email address is added as invisible metadata, as well as marked up as rel-me to facilitate [RelMeAuth](https://indieweb.org/RelMeAuth) via email.

For the last two I would’ve preferred to use `link` elements, but it turns out microformats don’t accept those.

Current output from a microformats parser:

```json
{
    "type": [
        "h-card"
    ],
    "properties": {
        "adr": [
            {
                "type": [
                    "h-adr"
                ],
                "properties": {
                    "country-name": [
                        "Singapore"
                    ],
                    "name": [
                        "Singapore"
                    ]
                },
                "value": "Singapore"
            }
        ],
        "name": [
            "William Narmontas"
        ],
        "role": [
            "Software Architect"
        ],
        "url": [
            "https://github.com/ScalaWilliam",
            "https://medium.com/@ScalaWilliam",
            "https://twitter.com/ScalaWilliam",
            "https://www.linkedin.com/in/scalawilliam/",
            "https://www.scalawilliam.com/"
        ],
        "photo": [
            "https://www.scalawilliam.com/index/images/headshot-junefourteen.jpg"
        ],
        "email": [
            "mailto:hello@scalawilliam.com"
        ]
    }
}
```

### CODE REVIEW

**Comment by Zegnat** [comment]
To get rid of [the invisible metadata (antipattern)](https://indieweb.org/antipatterns#invisible_metadata) the best solution is to somehow offer a different header to the homepage than to subpages. That way the first `div` (`id="william"`) block could be made part of the header and of the h-card.

That would also allow the Apt Elements link to be added to your h-card as an `org`.

Reworking the template system is something I found out of scope of the original issue though.

File: shared/template.html [hunk 1]
```diff
@@ -57,8 +57,10 @@
             </svg>
         </a>
         </nav>
-        <h1><a href="/">William Narmontas</a></h1>
-        <h2>Software Architect &bull; Singapore</h2>
+        <h1><a href="/" class="p-name u-url" rel="me">William Narmontas</a></h1>
```
Comment (ScalaWilliam) [inline shared/template.html]: Should href in this case be global and match the URL?

**Comment by ScalaWilliam** [comment]
Very nice.

Since you're effectively wanting to create a new issue, you *may* use the gitchiu bot if you like: https://twitter.com/ScalaWilliam/status/891450301152088064

I need to document it (#50, #51) properly though.

You can create an empty commit with `git commit --allow-empty`.

Example commit: https://github.com/ScalaWilliam/git-work/commit/3c59081bf835febbdcbfa946da6c4778fe513610
which creates this issue upon merge to master: https://github.com/ScalaWilliam/git-work/issues/25

File: shared/template.html (hunk 1 above)
Comment (Zegnat) [inline shared/template.html]: The URL there is meant to be “the link to the homepage of William Narmontas”. Wherever we are on the site, `/` will always link to your homepage. So this should be correct.

**Comment by Zegnat** [comment]
Do you want to create a new issue for redoing the template before merging this?

I am happy to write it, it just might not make a whole lot of sense.

**Pull request merged**